SQF/dayz_code/stringtable.xml

```diff
@@ -11279,13 +11279,16 @@
 			<Czech>Musíte mít syrové maso</Czech>
 		</Key>
 		<Key ID="STR_EPOCH_PLAYER_19">
-			<English>Wrong code entered too many times, wait 10 seconds</English>
+			<English>You entered the wrong code, wait %1 seconds.</English>
 			<German>Es wurde zu oft der falsche Code eingetippt. Du musst nun 10 Sekunden warten</German>
```

Comment:
For the german string:

Du hast einen falschen Code eingegeben, warte %1 Sekunden.